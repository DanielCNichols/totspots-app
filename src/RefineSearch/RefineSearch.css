.refine_form {
  display: block;
  width: 80%;
  margin: 100px auto;
}

.refine_form_label {
  text-align: center;
  display: block;
  margin: 1.5em;
}

.refine_form_select {
  display: block;
  width: 80%;
  padding: 5px 10px;
  height: 2.5rem;
  border-radius: 5px;
  margin: 5px auto;
  cursor: pointer;
}

.refine_form_button {
  display: block;
  min-width: 140px;
  height: 40px;
  margin: 20px auto;
  border-radius: 17px;
  border-color: blue;
  cursor: pointer;
  background-color: transparent;
  font-size: 1.1em;
}


@media screen and (min-width: 450px) {
  .refine_form {
    width: 70%;
  }
}