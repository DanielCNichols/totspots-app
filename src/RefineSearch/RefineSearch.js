import React from 'react'

export default function RefineSearch() {
  return (
     <section className="filter-search">
    <button>Refine search</button>
    <p>Note: This form will display when "refine search" is clicked</p>
      <form action="">
          <label htmlFor="Search">New Search
            <input type="text" placeholder="Search"/>
          </label>
          <label htmlFor="Sort">Sort by:
            <select name="Sort" id="Sort">
              <option value="">select</option>
              <option value="overall rating">Overall Rating</option>
              <option value="price">Price</option>
            </select>
          </label>
          <label htmlFor="Filter">Filter results by: 
            <select name="filter" id="filter">
              <option value="">Select</option>
              <option value="type">type</option>
              <option value="">Minimum Rating</option>
              <option value="">Amenities</option>
            </select>
          </label>
          <button>Search</button>
          <button>Cancel</button>
        </form>
  </section>
  )
}