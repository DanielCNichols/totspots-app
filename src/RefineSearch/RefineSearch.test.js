import React from 'react';
import ReactDOM from 'react-dom';
import RefineSearch from './RefineSearch.js';
import renderer from 'react-test-renderer';

it('renders RefineSearch without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<RefineSearch />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders RefineSearch as expected', () => {
  const tree = renderer.create(<RefineSearch name='RefineSearch' />).toJSON();
  expect(tree).toMatchSnapshot();
});
