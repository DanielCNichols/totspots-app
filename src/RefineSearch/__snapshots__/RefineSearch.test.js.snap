// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders RefineSearch as expected 1`] = `
<form
  className="refine_form"
  onSubmit={[Function]}
>
  <label
    className="refine_form_label"
    htmlFor="sort"
  >
    Sort results by:
    <select
      className="refine_form_select"
      name="sort"
    >
      <option>
        Select
      </option>
      <option
        value="avgPrice"
      >
        Price (low to high)
      </option>
      <option
        value="avgRating"
      >
        Rating(high to low)
      </option>
      <option
        value="avgVolume"
      >
        Volume Level (low to high)
      </option>
    </select>
    <button
      className="refine_form_button"
      type="submit"
    >
      Sort
    </button>
  </label>
</form>
`;
